mne/viz/_3d.py

```diff
@@ -1085,7 +1085,7 @@ def plot_alignment(info=None, trans=None, subject=None, subjects_dir=None,
         renderer.tube(origin=origin, destination=destination)
 
     renderer.set_camera(azimuth=90, elevation=90,
-                        distance=0.6, focalpoint=(0., 0., 0.))
+                        distance=0.6)
```

Comment:
The real question is what should the default actually be? I think (0., 0., 0.) is a reasonable choice for us. In all 3D plotting cases I can think of, (0., 0., 0.) is meaningful (center of some coordinate frame) so I would leave this as the default, and then just set it explicitly to `focalpoint=None` or `focalpoint=(...)` in places where we want it to be something different.

Reply:
We want a fixed value for `plot_alignment`? The default uses the bounds for now.

Reply:
Yes for `plot_alignment` in particular it seems reasonable to use `(0., 0., 0.)` because it's the alignment in the given `coord_frame`